lib/core/form/components/form.component.html

```diff
@@ -30,7 +30,7 @@ <h4 *ngIf="isTitleEnabled()">
         </mat-card-content>
         <mat-card-actions *ngIf="form.hasOutcomes()" class="adf-form-mat-card-actions">
             <!--[class.mdl-button--colored]="!outcome.isSystem"-->
-            <button *ngFor="let outcome of form.outcomes" 
+            <button [id]="'adf-form-'+ outcome.name  | lowercase" *ngFor="let outcome of form.outcomes"
```

Comment:
Maybe we need sanitising the outcome name for some characters that are invalid in CSS?

Reply:
<img width="985" alt="screen shot 2018-02-15 at 09 57 11" src="https://user-images.githubusercontent.com/1030050/36250758-a4e0520a-1236-11e8-9ddb-e4b13077f7a0.png">
seems all fine

Reply:
<img width="646" alt="screen shot 2018-02-15 at 09 57 56" src="https://user-images.githubusercontent.com/1030050/36250788-b84060b0-1236-11e8-8a91-131161a924ef.png">